### PR TITLE
Add installation instructions and 10 steps to follow to make a real run with QUAIL with example commands and sample output from stdout

### PR DESCRIPTION
**Pull request by senrabc**
I believe this start of a more detailed README.md will help with uptake and usage of QUAIL. QUAIL worked perfectly but I had to combine what I knew about honeyguide and use it to make a HOWTO for using quail. All 10 steps work.  Today I was able to pull down all of HCV prod and create a useful sqlite3 db for reporting.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ venv
 __pycache__
 quail.conf.yaml*
 *.db
+/examples/hcvprod_pull/DANGER
```
Comment (PFWhite) [inline .gitignore:7]: will this be read as an absolute path? maybe we should just disinclude anything with DANGER?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,347 @@
...
+    quail redcap generate ( <quail.conf.yaml> <project_name> <token> <url> ) [-i --initialize]
+    quail redcap get_meta (<project_name>) [ -q <quail.conf.yaml> ]
+    quail redcap get_data (<project_name>) [ -q <quail.conf.yaml> ]
+    quail redcap gen_meta (<project_name>) [ -q <quail.conf.yaml> ]
+    quail redcap gen_data (<project_name>) [ -q <quail.conf.yaml> ]
+    quail redcap make_import_files (<project_name>) [ -q <quail.conf.yaml> ]
+
+
+    ```
+  3. Goto ` $ cd examples/hcvprod_pull/DANGER ` folder. You may have to make
```
Comment (PFWhite) [inline README.md:41]: Maybe we should suggest that nothing should go in the code repository. That dodges a lot of the issues that could happen with PHI

File: README.md [hunk 3]
```diff
@@ -1,3 +1,347 @@
...
+  \___\_\\____/_/    \_\_____|______|
+
+   Q.U.A.I.L. Universal Analytics of Informatic Linkages
+
+
+```
+
+# Installation
+
+ 1. Run ` $ python setup.py install `
```
Comment (PFWhite) [inline README.md:26]: This worked? This is a python module that was meant to be `pip install ./PATH/TO/REPO` to get it to work.

**Review by PFWhite — COMMENTED** [review]
This is great thank you for the extensive documentation and good instructions!

File: .gitignore (hunk 1 above)
Comment (senrabc) [reply .gitignore:7]: I think maybe I was vague. It was just supposed to be an example.  I wrote this in step 3 " Goto $ cd examples/hcvprod_pull/DANGER folder. You may have to make this folder structure." telling them they may have to make these folders, as i did, to get the example to work.

File: README.md (hunk 2 above)
Comment (senrabc) [reply README.md:41]: caveat emptor:

I wrote: "This is up TO You to make sure you don't commit live data to github." in step 4

File: README.md (hunk 3 above)
Comment (senrabc) [reply README.md:26]: yep

Comment (senrabc) [reply README.md:26]: Ahh. Ok. I understand. Please feel free to fix any of my hackiness with your preferred methods.

**Comment by KevinHanson** [comment]
I was able to setup a vagrant, install the dependencies and environment, follow the instructions and create sqlite databases for the ADRC project in my DANGER folder since I don't have a token for HCV UNC.  Nice writeup!

**Pull request merged**